### PR TITLE
plugins/indent-blankline: swith to mkNeovimPlugin

### PR DESCRIPTION
**Pull request by GaetanLepage**

### CODE REVIEW

File: plugins/utils/indent-blankline.nix [hunk 1]
```diff
@@ -5,87 +5,48 @@
...
+    luaName = "ibl";
+    defaultPackage = pkgs.vimPlugins.indent-blankline-nvim;
+
+    maintainers = [maintainers.GaetanLepage];
+
+    # TODO introduced 2024-03-10: remove 2024-05-10
+    deprecateExtraOptions = true;
+    optionsRenamedToSettings = [
+      "debounce"
+      "viewportBuffer"
```
Comment (traxys) [inline plugins/utils/indent-blankline.nix:21]: does this work with  the suboptions (viewportBuffer.min, viewportBuffer.max) ?

**Review by traxys — CHANGES_REQUESTED** [review]

File: plugins/utils/indent-blankline.nix (hunk 1 above)
Comment (GaetanLepage) [reply plugins/utils/indent-blankline.nix:21]: The warning is triggered for the root option.
Maybe it is better to specify for each of the suboptions still.

Comment (traxys) [reply plugins/utils/indent-blankline.nix:21]: I'd find this more re-assuring  (and more helpful too)

Comment (GaetanLepage) [reply plugins/utils/indent-blankline.nix]: I agree, done.

**Review by traxys — APPROVED** [review]

**Pull request merged**